guide/how-it-works/payment-request-formats.md

```diff
@@ -19,119 +19,235 @@ images:
+   retina = "/assets/images/guide/how-it-works/payment-request-formats/adresses@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/payment-request-formats/adresses-mobile.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/payment-request-formats/adresses-mobile@2x.jpg"
+   alt-text = "How LSPs offer liquidity as a service"
+   width = 800
+   height = 348
+%}
 
-{% include fact/open.html color="blue" label="LNURL-Withdraw" icon="triangle" %}
+### Payment code
 
-<span style="word-break: break-word;">LNURL1DP68GURN8GHJ7MRWW4EXCTT5DAHKCCN00QHXGET8WFJK2UM0VEAX2UN09E3K7MF0W5LHZ0FHXSMRSVNXVESKVEPEX9NRGETYX5ENGVTYXCUR2ENYVE3KVDFKXYCXVDF4XCURSVR9XFJR2DMPX9NRVETPXPSKVCTPX3SKGEPCXGCRYKNQXCW</span>
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/how-it-works/payment-request-formats/payment-code.jpg"
+   retina = "/assets/images/guide/how-it-works/payment-request-formats/payment-code@2x.jpg"
+   alt-text = "A QR code of an offer"
+   width = 400
+   height = 500
+   layout = "float-right-desktop"
+%}
 
-{% include fact/close.html %}
+A payment code is a static payment request type that can receive multiple payments. The technical details of how these work is defined in [BIP 47](https://bips.xyz/47).
 
-{% include fact/open.html color="blue" label="LNURL-Auth" icon="triangle" %}
+Payment codes have been popularised in wallets such as [Samourai](https://samouraiwallet.com/) and [Sparrow](https://sparrowwallet.com/). These wallets abstract away the payment codes into a more user friendly payment addresses called [PayNyms](https://paynym.is/).
 
-<span style="word-break: break-word;">LNURL1DP68GURN8GHJ7MRWW4EXCTT5DAHKCCN00QHXGET8WFJK2UM0VEAX2UN09E3K7MF0W5LHGCT884KX7EMFDCNXKVFAVGURWWT9XYCNYE35XUMRXCEHVY6NGDEHVSUNWWPEV5UN2VMY8Q6RGDF4XVEXYE33V56NSETRVVMRGEFEVCMRWENPXCUXYEP48PNX2EQPG3G5S</span>
+A downside to payment codes is they require an additional on-chain transaction each time two users want to connect and use them.
 
-{% include fact/close.html %}
+</div>
 
-{% include fact/open.html color="blue" label="LNURL-Channel" icon="triangle" %}
+### Silent payment
 
-<span style="word-break: break-word;">LNURL1DP68GURN8GHJ7MRWW4EXCTT5DAHKCCN00QHXGET8WFJK2UM0VEAX2UN09E3K7MF0W5LHZ0F4VFJK2CEHX5CR2DT9XP3XZWP4V5EN2VF489SKYCFJX5UR2VTYV4SKYWRZX93RSEPJV9SKGVRRXYER2WT9XEJNZEPKX4NRGDEEXS6R2XNW3D6</span>
+Silent payments is a [newly proposed format](https://gist.github.com/RubenSomsen/c43b79517e7cb701ebf77eec6dbb46b8?permalink_comment_id=4113680) that allows two users to privately make an on-chain payment between them. This is similar to payment codes however does not need to establish a relationship between users limiting their on-chain footprint.
+
+The trade-off that silent payments require is receivers of silent payments require extra bandwidth to scan the bitcoin UTXO set to find payments made to them. This could be problematic on mobile and other less powerful devices.
```

Comment:
```suggestion
The trade-off that silent payments require is that receivers of silent payments need extra bandwidth to scan the bitcoin UTXO set, to find payments made to them. This could be problematic on mobile and other less powerful devices.
```